src/app/tabs/debugger/debuggerUI.js

```diff
@@ -103,38 +103,65 @@ class DebuggerUI {
     return this.isActive
   }
 
-  startDebugging (blockNumber, txNumber, tx) {
+  async getDebugWeb3 () {
+    return new Promise((resolve, reject) => {
+      executionContext.detectNetwork((error, network) => {
+        let web3
+        if (error || !network) {
+          web3 = init.web3DebugNode(executionContext.web3())
+        } else {
+          var webDebugNode = init.web3DebugNode(network.name)
+          web3 = (!webDebugNode ? executionContext.web3() : webDebugNode)
```

Comment:
Would this work ? `web3 = webDebugNode || executionContext.web3()`

Reply:
yeah.. I have a standard error if i do the other way around